packages/react-components/src/templates/Layout.tsx

```diff
@@ -1,65 +1,179 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useRouteMatch } from 'react-router-dom';
 import css from '@emotion/css';
 
-import { Header, Navigation } from '../molecules';
-import { smallDesktopScreen } from '../pixels';
+import { steel, paper } from '../colors';
+import { tabletScreen } from '../pixels';
+import { MenuHeader, MainNavigation, UserNavigation } from '../organisms';
+import { UserMenuButton } from '../molecules';
+import { Overlay } from '../atoms';
 
-const showNavigationQuery = `@media (min-width: ${smallDesktopScreen.width}px)`;
+const drawerQuery = `@media (max-width: ${tabletScreen.width - 1}px)`;
+const crossQuery = `@media (min-width: ${tabletScreen.width}px)`;
 
 const styles = css({
```

Comment:
This is about as small as this component can get unfortunately, I've already extracted out everything that isn't tightly coupled to the grid layout